.github/workflows/besu_expressapi.yaml

```diff
@@ -0,0 +1,65 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Build express app docker image triggered by :
+#  - push on develop branch for latest tags 
+
+name: Build express app docker image and push to ghcr
+on:
+  workflow_dispatch:
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - develop
+    paths:
+      - '.github/workflows/besu_expressapi.yaml'
+      - 'examples/supplychain-app/besu/express_nodeJS/Dockerfile'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  # on push to develop branch, create docker tags latest and jdk8-latest
+  expressapi-build-latest:
```

Comment:
We need 2 jobs.
One for `express-app-latest` and another for `release`
Check this file: https://github.com/hyperledger/bevel/blob/main/.github/workflows/quorum_images_build.yml

Reply:
Done